agent/testagent.go

```diff
@@ -472,17 +474,72 @@ func TestACLConfig() string {
 	`
 }
 
+const (
+	TestDefaultMasterToken      = "d9f05e83-a7ae-47ce-839e-c0d53a68c00a"
+	TestDefaultAgentMasterToken = "bca580d4-db07-4074-b766-48acc9676955'"
+)
+
+type ACLConfigParams struct {
+	PrimaryDatacenter string
+	DefaultPolicy     string
+	MasterToken       string
```

Comment:
Maybe add an AgentToken here if you forsee a situation where you'd want to test that behavior without using the master token?

Reply:
I figured if/when we wanted that it would be simple enough to add more to the template and params struct. For the time being this was enough for my purposes.